README.md

```diff
@@ -26,7 +26,7 @@ When the reader has completed this Code Pattern, they will understand how to:
 
 ## Included components
 
-* [Watson Visual Recognition](https://www.ibm.com/watson/developercloud/visual-recognition.html): Visual Recognition understands the contents of images - visual concepts tag the image, find human faces, approximate age and gender, and find similar images in a collection.
+* [Watson Visual Recognition](https://www.ibm.com/watson/developercloud/visual-recognition.html): Visual Recognition understands the contents of images - visual concepts: tag the image, find human faces, approximate age and gender, and find similar images in a collection.
```

Comment:
I think this needs more than a colon to fix it. Here's something from the website that looks good:
```
Visual Recognition understands the contents of images. Analyze images for scenes, objects, faces, colors, food, text, explicit content, and other subjects that can give you insights into your visual content.
``